blog/2023-07-13-reproducibility-designer.md

```diff
@@ -0,0 +1,41 @@
+---
+author: Antonin Delpeuch
+title: "Designer seeked for OpenRefine's reproducibility improvements"
```

Comment:
```suggestion
title: "Designer sought for OpenRefine's reproducibility improvements"
```
although using active voice instead of passive would be even better e.g. We're hiring a workflow designer!

Reply:
Active voice makes sense, but I would not qualify the position as a "workflow designer", because that feels a bit too narrow (there are reproducibility issues in scope which are not strictly about the workflow language). Also I think it could scare off people who would be good matches for the role.